docs/usage/cli/filter.rst

```diff
@@ -13,16 +13,24 @@ augur filter
     :prog: augur
     :path: filter
 
-How we subsample sequences in the zika-tutoral
-==============================================
+Guides
+======
+
+Below are some examples of using ``augur filter`` to sample data.
+
+Filtering
+---------
 
-As an example, we'll look that the ``filter`` command in greater detail using material from the :doc:`Zika tutorial <docs.nextstrain.org:tutorials/creating-a-workflow>`.
 The filter command allows you to selected various subsets of your input data for different types of analysis.
 A simple example use of this command would be
 
 .. code-block:: bash
 
-	augur filter --sequences data/sequences.fasta --metadata data/metadata.tsv --min-date 2012 --output filtered.fasta
+  augur filter \
+      --sequences data/sequences.fasta \
+      --metadata data/metadata.tsv \
+      --min-date 2012 \
+      --output filtered.fasta
```

Comment:
I see that the old docs only output sequences with the older `--output` argument, but since we're updating this section of the docs substantially now anyway, I'd recommend changing this example to use `--output-sequences` and also to include the `--output-metadata` argument. This example would maintain the conceptual alignment of two inputs and two outputs.

Reply:
Done: ae570a2451e1a66fe1373e204f8a7d0fade654fd